packages/react/src/components/Icon/Icon.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import IbmLogo from './svg/ibm-logo.svg';
```

Comment:
We should sync-up on how we want to manage the logo. Maybe not necessary to do for this pull request, but in my pull request #51 manages it slightly differently (part of that is because we were doing this in tandum).

I think it belongs in the `@carbon/icons` repo, but talking shortly with Pete who sits next to me from Brand there could potentially be concerns about including it within an open-source code base. Whether that's our codebase or Carbon's.

Maybe we could create an issue around this?

Reply:
Sounds good, @photodow. I figured it was deliberately left out, since it's such a glaring omission from the `icons` package.